examples/app/polymer/polymer.jsf.js

```diff
@@ -0,0 +1,39 @@
+var app = angular.module('makeithappen');
+app.run(['RenderService', 'PathResolver', '$rootScope', function(RenderService, PathResolver, $rootScope) {
+
+    function PolymerControl() {
+
+        var instance;
+        var path;
+
+        // set-up event binding
+        document.addEventListener('WebComponentsReady', function() {
```

Comment:
Me neither, but I couldn't think of another way. I think this is where Angular 2 would probably be more useful since we could bind against value and the respective event.